.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - 2.7
+  - 3.5
```

Comment:
It's nice to keep the project moving to Python 3, but not dropping Python 2 support at once. so I suggest we keep `2.7` here to run tests with both `2.7` and `3.5`